ext/cgi/escape/escape.c

```diff
@@ -63,7 +63,7 @@ optimized_escape_html(VALUE str)
 
     if (RSTRING_LEN(str) < (dest - buf)) {
         *dest = '\0';
```

Comment:
This assignment is no longer needed.

Reply:
I hoped so, but I was too lazy to check it. Thanks :bow: https://github.com/ruby/ruby/pull/2226/commits/4d23502dc6b74ffecbb8914cb5db2d21a36b6d84